services/forms-web-app/package.json

```diff
@@ -27,7 +27,8 @@
     "http-errors": "~1.6.3",
     "morgan": "~1.9.1",
     "node-sass": "^4.14.1",
-    "nunjucks": "^3.2.2"
+    "nunjucks": "^3.2.2",
+    "pino": "^6.7.0"
   },
   "devDependencies": {
     "@semantic-release/changelog": "^5.0.1",
```

Comment:
It might be worth adding `pino-pretty` dependency and changing the `start:dev` script to:

`nodemon src/server.js | pino-pretty -tlc`

This will mean I can collate the logs in the deployment, but you can see the logs nicely in dev. Happy to approve without this though

Reply:
Added pino-pretty and updates start:dev script.